### PR TITLE
Adding simple CreateVhdx method

### PR DESCRIPTION
**Pull request by swernli**
This adds a simple CreateVhdx method, but also provides the foundations for later vhd related wrapper functions.
@jstarks @jhowardmsft

### CODE REVIEW

**Comment by jstarks** [comment]
Please split mksyscall submission into two commits.

File: vhd/vhd.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+const createVirtualDiskFlagDoNotCopyMetadataFromParent uint32 = 4
+
+type createVirtualDiskParameters struct {
+	Version  uint32 // Must always be set to 2
+	Version2 struct {
+		UniqueID                 [16]byte
+		MaximumSize              uint64
+		BlockSizeInBytes         uint32
+		SectorSizeInBytes        uint32
+		ParentPath               string
```
Comment (jstarks) [inline vhd/vhd.go]: must be *uint16

File: vhd/vhd.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+	}
+}
+
+// CreateVhdx will create a simple vhdx file at the given path using default values.
+func CreateVhdx(path string, maxSizeInGb, blockSizeInMb uint32) error {
+	var defaultType virtualStorageType
+
+	parameters := createVirtualDiskParameters{
+		Version: 2,
+	}
```
Comment (jstarks) [inline vhd/vhd.go]: initialize this as one thing

File: vhd/vhd.go [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+		virtualDiskAccessNONE,
+		nil,
+		createVirtualDiskFlagNone,
+		0,
+		&parameters,
+		nil,
+		&handle); err != nil {
+		return err
+	}
+
```
Comment (jstarks) [inline vhd/vhd.go]: forgot to close the handle

Comment (swernli) [inline vhd/vhd.go]: Added.

**Comment by jstarks** [comment]
I guess I expected to see the first commit just being the add of mksyscall, and the second commit being the modify of mksyscall and the addition of the rest of the code...

**Comment by swernli** [comment]
@jstarks Ok, finally got this into the right state.

**Pull request merged**